benchflow-dsl/src/main/scala/cloud/benchflow/dsl/definition/configuration/goal/explorationspace/explorationvalues/ExplorationValuesIntYamlProtocol.scala

```diff
@@ -62,10 +72,48 @@ object ExplorationValuesIntYamlProtocol extends DefaultYamlProtocol {
 
   }
 
-  def getList(range: List[Int]): List[Int] = {
+  private def getRangeList(range: List[Int]): Try[List[Int]] = {
+
+    if (range.size != 2) {
+
+      Failure(new BenchFlowDeserializationException("range must be specified as a List of 2 elements", new Throwable))
 
-    (range.head to range(1)).toList
+    } else {
+
+      Success((range.head to range(1)).toList)
```

Comment:
Is this the default for a range? `+1` as step? If yes, make it clear in a comment.